libs/community/langchain_community/llms/oci_data_science_model_deployment_endpoint.py

```diff
@@ -256,33 +599,26 @@ def _llm_type(self) -> str:
     def _default_params(self) -> Dict[str, Any]:
         """Get the default parameters for invoking OCI model deployment TGI endpoint."""
         return {
+            "model": DEFAULT_MODEL_NAME,  # can be any
             "best_of": self.best_of,
             "max_new_tokens": self.max_tokens,
```

Comment:
Looks like the new TGI API spec also uses `max_tokens` instead of `max_new_tokens`.
https://huggingface.github.io/text-generation-inference/#/Text%20Generation%20Inference/completions

Reply:
fixed.